### DISCUSSION TITLE
How put the color in chart with Xlsx

### DISCUSSION THREAD

**Discussion by obrienphao** [QtExcel/QXlsx#224] [Ideas]
Hello,
I came here ask is possible to put the color in chart with Xlsx, i have my chart, but i have not find the methode for change the color of chart if possible the help me Example:  

           ` QXlsx::Document xlsx("myExcel.xlsx");
             chart* histo->xlsx.insertChart(8, 8, QSize(300, 300));
             histo->setChartType(Chart::CT_BarChart);
             histo->setChartTitle("Conso");
             histo->addSeries(CellRange("B2:B10")); 
             histo->addSeries(CellRange("D2:D10")); 
             histo->addSeries(CellRange("C2:C10"));`
           
how put the color in different series ?

**Comment by j2doll** [comment]
Dear @obrienphao 

Sorry for late reply.

It seems like it's not a supported feature (maybe). 

Can you show me an Excel file (or image) that represents the feature you mentioned?
